File: go-gqlgen-gorm-directive/script/seed.go
//go:build ignore
// +build ignore

package main

import (
	"fmt"
	"go-gqlgen-gorm-directive/model"
	"log"

	"gorm.io/driver/sqlite"
	"gorm.io/gorm"
)

func main() {
	db, err := gorm.Open(sqlite.Open("gorm.db"), &gorm.Config{})
	if err != nil {
		log.Fatal(err)
	}

	for u := 1; u <= 10; u++ {
		user := &model.User{
			Name:     fmt.Sprintf("user%d", u),
			Email:    fmt.Sprintf("user%d@example.com", u),
			Password: "password",
		}
		db.Create(&user)

		for p := 1; p <= 10; p++ {
			db.Create(&model.Post{
				Title:   fmt.Sprintf("title%d", p),
				Content: fmt.Sprintf("content%d", p),
				UserID:  user.ID,
			})
		}
	}
}

File: ts-schema-first/packages/server/prisma/seed.ts
import { PrismaClient } from "@prisma/client"

async function seed() {
  const prisma = new PrismaClient()

  for (let i = 1; i <= 10; i++) {
    await prisma.user.create({
      data: {
        name: `User ${i}`,
        email: `user${1}@example.com`,
      },
    })
  }
}

seed()
